include/mimick/mock.h

```diff
@@ -46,7 +46,10 @@ struct mmk_params {
     struct mmk_params *next;
 };
 
-struct mmk_mock_options {
+// Force bit-field alignment to circumvent a bug in GCC
+// versions 6, 7, and 8, and a fixup notice on GCC 9.
+// See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=88469.
+struct __attribute__((aligned(32))) mmk_mock_options {
```

Comment:
From https://gcc.gnu.org/onlinedocs/gcc-9.3.0/gcc/Common-Type-Attributes.html#Common-Type-Attributes, my understanding is that the `aligned` attribute is in bytes.  Do you really mean to align it to 32-bytes?

Also, is this going to work when compiling with clang/MSVC?

Reply:
> Do you really mean to align it to 32-bytes?

I meant 4 bytes... :man_facepalming: 

> Also, is this going to work when compiling with clang/MSVC?

Nope... :man_facepalming: 

That's what happens when you do things in a rush and half-burned.

Reply:
See https://github.com/ros2/Mimick/pull/10/commits/a95e17499a595bb8400162bc03728f4ab534e5c8